hotness/buildsys.py

```diff
@@ -29,6 +47,12 @@ def __init__(self, consumer, config):
         self.ca_cert = config['ca_cert']
         self.git_url = config['git_url']
         self.userstring = config['userstring']
+        if isinstance(self.userstring, six.string_types):
+            msg = ('Using a string for "userstring" in "hotness.koji"'
+                   ' is deprecated, please use a tuple')
+            warnings.warn(msg, DeprecationWarning)
+        else:
+            self.userstring = ' '.join(self.userstring)
```

Comment:
Yeah, that would make more sense. I kept changing my mind about which one I wanted. I'm still not sure which one I want, but I guess string is easier.

There's a problem where rebase-helper doesn't work because no author is set for the repo (and then the commit fails) and so my motivation for having it be a tuple was that it'd be easier to set the author and email fields. It's probably just as easy to split on ``<`` though. Although I guess someone might want ``<`` in their name (or some reason).